config/routes.rb

```diff
@@ -236,6 +236,7 @@
   resources :podcasts, only: %i[new create]
   resources :article_approvals, only: %i[create]
   resources :video_chats, only: %i[show]
+  resources :user_subscriptions, only: %i[create]
```

Comment:
I will likely exclude this route and maybe more from the service worker in a later PR once we sort out exactly how we're going to do the author's view.